recipes/_configure_server.rb

```diff
@@ -55,8 +58,10 @@
     server_stanzas[stanza] = {}
     server_stanzas[stanza]['master_uri'] = master_uri
     server_stanzas[stanza]['pass4SymmKey'] = CernerSplunk::ConfTemplate.compose encrypt_password, CernerSplunk::ConfTemplate::Value.constant(value: pass) unless pass.empty?
-    # Until we support multisite clusters, set multisite explicitly false
-    server_stanzas[stanza]['multisite'] = false
+    if CernerSplunk.multisite_cluster?(bag, cluster)
+      server_stanzas[stanza]['multisite'] = true
+      server_stanzas[stanza]['site'] = bag['site']
```

Comment:
So if you wanted to disable search affinity you would have to essentially duplicate the cluster data bag right? And likely the environment or role assigning it too?  It would be nice to be able to set the search head to site0 without having to do that. Possibly separate the indexer_site and search_site attributes in the multisite bags?

Reply:
> So if you wanted to disable search affinity you would have to essentially duplicate the cluster data bag right?

That is correct.

> Possibly separate the indexer_site and search_site attributes in the multisite bags?

I think this should be in the site databag and not the multisite databag. So that I can have a SH in site1 with search affinity disabled and another SH in site2 with SH enabled (not clustered though). Also, I'm not sure what _indexer_site_ is for.

Reply:
Another option I thought of was to add a configuration to the site data bag like "disable_search_affinity" that could optionally be set to true. That would trigger all search heads in the site to use site0 instead of the site listed in the data bag.

Reply:
Added disable_search_affinity attribute here: 8b304e88058a09d791a494d71bd8a479b69b8d12